client-registry-jempi/docker-compose.combined-dev.yml

```diff
@@ -21,3 +21,9 @@ services:
         target: 50000
         protocol: tcp
         mode: host
+
+  jempi-postgresql:
+    ports:
+      - published: 5432
+        target: 5435
```

Comment:
postgres' default port is 5432. The target should be 5432

Reply:
already other postgres port is 5432